docs/wire-protocol.txt

```diff
@@ -257,7 +284,7 @@ request protocol upgrade
 <1 byte protocol max version to upgrade to>
 <remaining bytes discarded>
 
-command: 'V' (version upgrade)
+command: 'v' (version upgrade)
```

Comment:
Is there a reason to change all the casings here? Is it to help migration?

Reply:
disctinction from inner and outer message commands, stylistic shtick.